ui/app/AppLayouts/Wallet/WalletUtils.qml

```diff
@@ -7,9 +7,36 @@ import StatusQ.Core.Theme 0.1
 
 import StatusQ.Core.Utils 0.1 as StatusQUtils
 
-import "stores" as WalletStores
+import AppLayouts.Wallet.stores 1.0 as WalletStores
 
 QtObject {
+
+    property QtObject _d: QtObject {
+        id: d
+
+        property var chainColors: ({})
+
+        function initChainColors(model) {
+            for (let i = 0; i < model.count; i++) {
+                const item = SQUtils.ModelUtils.get(model, i)
```

Comment:
This can't work under `SQUtils`, you imported it as `import StatusQ.Core.Utils 0.1 as StatusQUtils` on the top